fulp_modules/cult_changes/cult_changes.dm

```diff
@@ -0,0 +1,74 @@
+/// Cult stun
+/obj/item/melee/blood_magic/stun/afterattack(atom/target, mob/living/carbon/user, proximity)
+	if(!isliving(target) || !proximity)
+		return
+	var/mob/living/L = target
+	if(iscultist(target))
+		return
+	if(iscultist(user))
+		user.visible_message("<span class='warning'>[user] holds up [user.p_their()] hand, which explodes in a flash of red light!</span>", \
+							"<span class='cultitalic'>You attempt to stun [L] with the spell!</span>")
+
+		user.mob_light(_color = LIGHT_COLOR_BLOOD_MAGIC, _range = 3, _duration = 2)
+
+		var/anti_magic_source = L.anti_magic_check()
+		if(anti_magic_source)
+
+			L.mob_light(_color = LIGHT_COLOR_HOLY_MAGIC, _range = 2, _duration = 100)
+			var/mutable_appearance/forbearance = mutable_appearance('icons/effects/genetics.dmi', "servitude", -MUTATIONS_LAYER)
+			L.add_overlay(forbearance)
+			addtimer(CALLBACK(L, /atom/proc/cut_overlay, forbearance), 100)
+
+			if(istype(anti_magic_source, /obj/item))
+				var/obj/item/ams_object = anti_magic_source
+				target.visible_message("<span class='warning'>[L] starts to glow in a halo of light!</span>", \
+									   "<span class='userdanger'>Your [ams_object.name] begins to glow, emitting a blanket of holy light which surrounds you and protects you from the flash of light!</span>")
+			else
+				target.visible_message("<span class='warning'>[L] starts to glow in a halo of light!</span>", \
+									   "<span class='userdanger'>A feeling of warmth washes over you, rays of holy light surround your body and protect you from the flash of light!</span>")
+
+		else
+			if(HAS_TRAIT(target, TRAIT_MINDSHIELD))
+				var/mob/living/carbon/C = L
+				to_chat(user, "<span class='cultitalic'>Their mind is too strong, resisting the spell, but it damaged them nonetheless!</span>")
+				C.stuttering += 8
+				C.dizziness += 20
+				C.Jitter(6)
+				C.bleed(30)
+				C.apply_damage(60, STAMINA, BODY_ZONE_CHEST)
```

Comment:
One of the major issues with stamina damage for cultists is that they generally don't have much of it at all. Typically, when they run into mindshielded people, their last resort is lethal attacks which is all brute and the like. This was the major problem with this same change on tg, where baton jousts ended up favouring anyone with a baton due to their followup on the first attack, which stun doesn't have given it expends the attack immediately after use.

Since this is causing a bleed effect, I might recommend instead having this deal a (reasonable) amount of brute and have wounding power instead, and maybe a knockdown to top it off. Once everyone is mindshield, it becomes incredibly easy to streamroll into a cult base with your effect new brain armor, and cultists can't differentiate a mindshielded target from a nonmindshelded target.

Reply:
> One of the major issues with stamina damage for cultists is that they generally don't have much of it at all. 

We dont want them using Stamina damage against Mindshielded people, afterall mindshielded folk gets their body permanently removed from the round if they lose.

> their last resort is lethal attacks which is all brute and the like.

Lethals on Mindshielded should be their first resort, the stamina damage is just to slow the user down so they can't escape at least, but maybe I can lower it.

> This was the major problem with this same change on tg, where baton jousts ended up favouring anyone with a baton due to their followup on the first attack, which stun doesn't have given it expends the attack immediately after use.

This is a fair point, I might remove the stamina damage entirely afterall.

> and cultists can't differentiate a mindshielded target from a nonmindshelded target.

Thats why I added sechuds into the zealot's